### PR TITLE
Remove discover implement embeddable dashboard on aws module

### PR DESCRIPTION
**Pull request by yenienserrano**
## Description
implement the embeddable dashboard on AWS -> dashboard tab and deprecate any use of kibana-integrations components.
 
## Issues Resolved
- #6493 

## Evidence

image
image

## Test
# Global

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check data results consistency between events and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters persist between the events and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters are cleaned when throwing in the events tab | :black_circle: | :black_circle: | :black_circle: |
| Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs) | :black_circle: | :black_circle: | :black_circle: |
| Check pinned agent filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check the request with hidden filters applied (allow agents, hide manager alerts)  | :black_circle: | :black_circle: | :black_circle: |
| Check the fixer filter cannot be removed from the filter bar (the remove icon is hidden) | :black_circle: | :black_circle: | :black_circle: |

- Check data results consistency between events and dashboard tab

<details>

</details>

-  Check the filters persist between the events and dashboard tab

<details>

</details>

- Check the filters are cleaned when throwing in the events tab

<details>

</details>

- Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs)

<details>

</details>

- Check pinned agent filter on tabs

<details>

</details>

- Check the request with hidden filters applied (allow agents, hide manager alerts)

<details>

`Enable/disable the hideManagerAlerts settings on App Settings`

Screenshot 2024-03-31 at 12 29 47

</details>

- Check the fixed filter cannot be removed from the filter bar

<details>

</details>

# Dashboard tab

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use search bar input to filter any word and check if the dashboard shows the corresponding results | :black_circle: | :black_circle: | :black_circle: |
| Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter | :black_circle: | :black_circle: | :black_circle: |
| Add a filter through interaction with a visualization and check that the dashboard updates | :black_circle: | :black_circle: | :black_circle: |
| Use the input and type an incorrect word to check if the no results message is shown | :black_circle: | :black_circle: | :black_circle: |

- Use search bar input to filter any word and check if the dashboard shows the corresponding results

<details>

</details>

- Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter

<details>

</details>

- Add a filter through interaction with a visualization and check that the dashboard updates

<details>

</details>

- Use the input and type an incorrect word to check if the no results message is shown

<details>

</details>

# Events tab (Discover)

## Search bar

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Apply filters using search bar input | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the search bar `add filter` feature | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the date picker | :black_circle: | :black_circle: | :black_circle: |
| Apply filters with no results (Check the `No results match your search criteria` message)| :black_circle: | :black_circle: | :black_circle: |
| Use the `saved queries` feature (Check saved queries CRUD) | :black_circle: | :black_circle: | :black_circle: |

- Apply filters using search bar input

<details>

</details>

- Apply filters using the search bar `add filter` feature

<details>

</details>

- Apply filters using the date picker

<details>

</details>

- Apply filters with no results (Check the `No results match your search criteria` message)

<details>

</details>

- Use the `saved queries` feature (Check saved queries CRUD)

<details>

</details>

## Histogram chart

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check histogram chart hits consistency | :black_circle: | :black_circle: | :black_circle: |
| Use chart options (three points: maximize, minimize, inspect) | :black_circle: | :black_circle: | :black_circle: |

- Check histogram chart hits consistency 

<details>

</details>

- Use chart options (three points: maximize, minimize, inspect) 

<details>

</details>

## Data grid

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check all table header features (sort, show/hide columns, density, etc) | :black_circle: | :black_circle: | :black_circle: |
| Check table pagination and rows per page | :black_circle: | :black_circle: | :black_circle: |
| Check document detail flyout | :black_circle: | :black_circle: | :black_circle: |

- Check all table header features (sort, show/hide columns, density, etc)

<details>

</details>

- Check table pagination and rows per page

<details>

</details>

- Check document detail flyout

<details>

</details>

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Comment by JuanGarriuz** [comment]
## Test
# Global

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check data results consistency between inventory and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters persist between the inventory and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters are cleaned when throwing in the events tab | :black_circle: | :black_circle: | :black_circle: |
| Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs) | :black_circle: | :black_circle: | :black_circle: |
| Check rule.mitre.id filter on Events and Framework tab | :black_circle: | :black_circle: | :black_circle: |
| Check pinned agent filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check the request with hidden filters applied (allow agents, hide manager alerts)  | :black_circle: | :black_circle: | :black_circle: |
| Check the fixer filter cannot be removed from the filter bar (the remove icon is hidden) | :black_circle: | :black_circle: | :black_circle: |

- Check data results consistency between inventory and dashboard tab

<details>


</details>

-  Check the filters persist between the inventory and dashboard tab

<details>


</details>

- Check the filters are cleaned when throwing in the events tab

<details>


</details>

- Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs)

<details>


</details>

- Check rule.mitre.id filter on Events and Framework tab

<details>


</details>

- Check pinned agent filter on tabs

<details>


</details>

- Check the request with hidden filters applied (allow agents, hide manager alerts)

<details>

`Enable/disable the hideManagerAlerts settings on App Settings`

Screenshot 2024-03-31 at 12 29 47


</details>

- Check the fixed filter cannot be removed from the filter bar

<details>

</details>


# Dashboard tab

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use search bar input to filter any word and check if the dashboard shows the corresponding results | :black_circle: | :black_circle: | :black_circle: |
| Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter | :black_circle: | :black_circle: | :black_circle: |
| Add a filter through interaction with a visualization and check that the dashboard updates | :black_circle: | :black_circle: | :black_circle: |
| Use the input and type an incorrect word to check if the no results message is shown | :black_circle: | :black_circle: | :black_circle: |

- Use search bar input to filter any word and check if the dashboard shows the corresponding results

<details>


</details>

- Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter

<details>


</details>

- Add a filter through interaction with a visualization and check that the dashboard updates

<details>


</details>

- Use the input and type an incorrect word to check if the no results message is shown

<details>


</details>


# Events tab (Discover)

## Search bar

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Apply filters using search bar input | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the search bar `add filter` feature | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the date picker | :black_circle: | :black_circle: | :black_circle: |
| Apply filters with no results (Check the `No results match your search criteria` message)| :black_circle: | :black_circle: | :black_circle: |
| Use the `saved queries` feature (Check saved queries CRUD) | :black_circle: | :black_circle: | :black_circle: |

- Apply filters using search bar input

<details>


</details>

- Apply filters using the search bar `add filter` feature

<details>


</details>

- Apply filters using the date picker

<details>


</details>

- Apply filters with no results (Check the `No results match your search criteria` message)

<details>


</details>

- Use the `saved queries` feature (Check saved queries CRUD)

<details>


</details>


## Histogram chart

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check histogram chart hits consistency | :black_circle: | :black_circle: | :black_circle: |
| Use chart options (three points: maximize, minimize, inspect) | :black_circle: | :black_circle: | :black_circle: |

- Check histogram chart hits consistency 

<details>


</details>


- Use chart options (three points: maximize, minimize, inspect) 

<details>


</details>

## Data grid

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check all table header features (sort, show/hide columns, density, etc) | :black_circle: | :black_circle: | :black_circle: |
| Check table pagination and rows per page | :black_circle: | :black_circle: | :black_circle: |
| Check document detail flyout | :black_circle: | :black_circle: | :black_circle: |

- Check all table header features (sort, show/hide columns, density, etc)

<details>


</details>

- Check table pagination and rows per page

<details>


</details>


- Check document detail flyout

<details>


</details>

**Review by JuanGarriuz — DISMISSED** [review]

**Comment by jbiset** [comment]
## CR:


## Test UI:

# Global

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check data results consistency between events and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters persist between the events and dashboard tab | :black_circle: | :black_circle: | :black_circle: |
| Check the filters are cleaned when throwing in the events tab | :black_circle: | :black_circle: | :black_circle: |
| Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs) | :black_circle: | :black_circle: | :black_circle: |
| Check pinned agent filter on tabs | :black_circle: | :black_circle: | :black_circle: |
| Check the request with hidden filters applied (allow agents, hide manager alerts)  | :black_circle: | :black_circle: | :black_circle: |
| Check the fixer filter cannot be removed from the filter bar (the remove icon is hidden) | :black_circle: | :black_circle: | :black_circle: |

- Check data results consistency between events and dashboard tab

<details>


</details>

-  Check the filters persist between the events and dashboard tab

<details>


</details>

- Check the filters are cleaned when throwing in the events tab

<details>


</details>

- Check cluster-manager filter on tabs. Must be applied when the tab is shown (all tabs)

<details>


</details>

- Check pinned agent filter on tabs

<details>


</details>

- Check the request with hidden filters applied (allow agents, hide manager alerts)

<details>

`Enable/disable the hideManagerAlerts settings on App Settings`

Screenshot 2024-03-31 at 12 29 47


</details>

- Check the fixed filter cannot be removed from the filter bar

<details>

</details>


# Dashboard tab

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Use search bar input to filter any word and check if the dashboard shows the corresponding results | :black_circle: | :black_circle: | :black_circle: |
| Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter | :black_circle: | :black_circle: | :black_circle: |
| Add a filter through interaction with a visualization and check that the dashboard updates | :black_circle: | :black_circle: | :black_circle: |
| Use the input and type an incorrect word to check if the no results message is shown | :black_circle: | :black_circle: | :black_circle: |

- Use search bar input to filter any word and check if the dashboard shows the corresponding results

<details>


</details>

- Select an option from one of the recommended filters and check that the corresponding filter is added and the dashboard changes according to the applied filter

<details>


</details>

- Add a filter through interaction with a visualization and check that the dashboard updates

<details>


</details>

- Use the input and type an incorrect word to check if the no results message is shown

<details>


</details>


# Events tab (Discover)

## Search bar

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Apply filters using search bar input | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the search bar `add filter` feature | :black_circle: | :black_circle: | :black_circle: |
| Apply filters using the date picker | :black_circle: | :black_circle: | :black_circle: |
| Apply filters with no results (Check the `No results match your search criteria` message)| :black_circle: | :black_circle: | :black_circle: |
| Use the `saved queries` feature (Check saved queries CRUD) | :black_circle: | :black_circle: | :black_circle: |

- Apply filters using search bar input

<details>


</details>

- Apply filters using the search bar `add filter` feature

<details>


</details>

- Apply filters using the date picker

<details>


</details>

- Apply filters with no results (Check the `No results match your search criteria` message)

<details>


</details>

- Use the `saved queries` feature (Check saved queries CRUD)

<details>


</details>


## Histogram chart

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check histogram chart hits consistency | :black_circle: | :black_circle: | :black_circle: |
| Use chart options (three points: maximize, minimize, inspect) | :black_circle: | :black_circle: | :black_circle: |

- Check histogram chart hits consistency 

<details>


</details>


- Use chart options (three points: maximize, minimize, inspect) 

<details>


</details>

## Data grid

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Check all table header features (sort, show/hide columns, density, etc) | :black_circle: | :black_circle: | :black_circle: |
| Check table pagination and rows per page | :black_circle: | :black_circle: | :black_circle: |
| Check document detail flyout | :black_circle: | :black_circle: | :black_circle: |

- Check all table header features (sort, show/hide columns, density, etc)

<details>


</details>

- Check table pagination and rows per page

<details>


</details>


- Check document detail flyout

<details>


</details>

**Review by jbiset — APPROVED** [review]
LGTM!

File: plugins/main/public/components/overview/amazon-web-services/dashboards/styles.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.pci-dss-dashboard-responsive {
```
[the remark this answers is not in the document]
Comment (yenienserrano) [reply plugins/main/public/components/overview/amazon-web-services/dashboards/styles.scss]: Solved [23f8962](https://github.com/wazuh/wazuh-dashboard-plugins/pull/6494/commits/23f896222a9e4bfbfd7ee922824243d79bea845f)

**Review by JuanGarriuz — APPROVED** [review]

**Review by jbiset — APPROVED** [review]

**Pull request merged**